js/ui/speed_dial_action/speed_dial_item.js

```diff
@@ -38,10 +41,40 @@ const SpeedDialItem = Overlay.inherit({
         this.$element().addClass(FAB_CLASS);
         this.callBase();
         this._renderIcon();
+        this._renderLabel();
         this.option("useInkRipple") && this._renderInkRipple();
         this._renderClick();
     },
 
+    _renderLabel() {
+        !!this._$label && this._$label.remove();
+
+        const labelText = this.option("label");
+
+        if(!labelText) {
+            this._$label = null;
+            return;
+        }
+
+        const $element = $("<div>").addClass(FAB_LABEL_CLASS);
+
+        this._$label = $element
+            .html(labelText)
```

Comment:
Does it mean `label` works just like `template`? How about script tags injected in this string?